pkg/env/sensor.go

```diff
@@ -21,10 +21,6 @@ var (
 	// LocalImageScanningEnabled is used to specify if Sensor should attempt to scan images via a local Scanner.
 	LocalImageScanningEnabled = RegisterBooleanSetting("ROX_LOCAL_IMAGE_SCANNING_ENABLED", false)
 
-	// ForceLocalImageScanning expands the scope of `LocalImageScanningEnabled` to have sensor analyze all images
-	// using the local scanner instead of only images from OCP internal registries
-	ForceLocalImageScanning = RegisterBooleanSetting("ROX_FORCE_LOCAL_IMAGE_SCANNING", false)
```

Comment:
Not sure how you planned on handling this, but we should call it out in the changelog.

Reply:
This was replaced by the sync'd 'delegated registry config' - will update the changelog